frontend/components/product-list/sections/FilterableProductsSection/ProductList.tsx

```diff
@@ -71,128 +71,143 @@ export function ProductListItem({ product }: ProductListItemProps) {
             }}
             py="7"
             alignItems="flex-start"
+            justify="space-between"
             px="4"
          >
-            <Box
-               flexGrow={1}
-               flexShrink={0}
-               w={{
-                  base: '100px',
-                  sm: '160px',
-                  md: '140px',
-                  lg: '160px',
+            <Stack
```

Comment:
Why `Stack` and not `Flex`?

Reply:
@andyg0808 
Because `Stack` allows me to specify `spacing="4"`